LangCache: Move certain docs to develop/ai (#1798)
redis/docs: content/operate/rc/langcache/_index.md

- rc
description: Store LLM responses for AI applications in Redis Cloud.
hideListLinks: true
linktitle: LangCache
title: Semantic caching with LangCache on Redis Cloud
weight: 36
bannerText: LangCache on Redis Cloud is currently available as a public preview.
bannerChildren: true
---

> LangCache is a semantic caching service available as a REST API that stores LLM responses for fast and cheaper retrieval, built on the Redis vector database. By using semantic caching, customers can significantly reduce API costs and lower the average latency of their generative AI applications.

Reviewer: ```suggestion
LangCache is a semantic caching service available as a REST API that stores LLM responses for fast and cheaper retrieval, built on the Redis vector database. By using semantic caching, you can significantly reduce API costs and lower the average latency of your generative AI applications.
```
